pkg/gameservers/controller.go

```diff
@@ -521,6 +521,11 @@ func (c *Controller) syncGameServerCreatingState(ctx context.Context, gs *agones
 				})
 			}
 		}
+		if runtime.FeatureEnabled(runtime.FeatureAutopilotPassthroughPort) && portPolicyOption(gs, agonesv1.Passthrough) {
+			gs.Spec.Template.Labels = make(map[string]string)
+			gs.Spec.Template.Labels[agonesv1.GameServerPortPolicyPodLabel] = "autopilot-passthrough"
```

Comment:
I would expect this to be in the GKE `cloudproduct` so that it's just triggered on Autopilot.

Reply:
Okok moved implementation to gke.go inside a method call in `MutateGameServerPod()`